python/lsst/ts/electrometer/csc.py

```diff
@@ -220,6 +286,7 @@ async def do_setRange(self, data):
             await self.controller.set_range(set_range=data.setRange)
             await self.report_detailed_state(DetailedState.NOTREADINGSTATE)
         except Exception:
+            self.log.error("setRange failed.")
```

Comment:
Either use `log.exception` or capture the exception and include it in your log message. If a traceback is useful, use `log.exception`.